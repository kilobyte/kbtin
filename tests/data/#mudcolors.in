#mudcolors black blue green cyan maroon magenta brown gray charcoal azure lime aqua red pink yellow white
#run a //selfpipe
#de 2 #end
~0~ ~1~ ~2~ ~3~ ~4~ ~5~ ~6~ ~7~ ~8~ ~9~ ~10~ ~11~ ~12~ ~13~ ~14~ ~15~
