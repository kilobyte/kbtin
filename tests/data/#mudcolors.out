#outgoing color codes table initialized
black blue green cyan maroon magenta brown gray charcoal azure lime aqua red pink yellow white

#SESSION 'a' DIED.
